src/app/views/pages/_sessions.html.erb

```diff
@@ -1,4 +1,5 @@
-  <%= cache Session.maximum(:updated_at), expires_in: 20.seconds do %>
+  <%# Note that fragment contains user-specific info, must be cached per user %>
+  <%= cache [Session.maximum(:updated_at), current_participant], expires_in: 20.seconds do %>
```

Comment:
Here: https://github.com/minnestar/sessionizer/blob/master/src/app/views/sessions/_session.html.erb#L1

`attending_class(session)` gives results based on the current user.